jsettlers.common/src/main/resources/jsettlers/common/buildings/amazon/baker.xml

```diff
@@ -82,9 +82,9 @@
 	<buildmark dx="-2" dy="0" />
 	<buildmark dx="-2" dy="-3" />
 
-	<image file="14" for="GUI" type="GUI" sequence="39" image="0"/>
-	<image file="13" for="FINAL" type="SETTLER" sequence="23" image="0"/>
-	<image file="13" for="BUILD" type="SETTLER" sequence="23" image="1"/>
+	<image file="44" for="GUI" type="GUI" sequence="42" image="0"/>
+	<image file="43" for="FINAL" type="SETTLER" sequence="28" image="0"/>
+	<image file="43" for="BUILD" type="SETTLER" sequence="28" image="1"/>
```

Comment:
This no longer fits their blocked tiles, does it?
I think logical correctness (people aren't walking through buildings and settlers are leaving/entering where doors are) is more important than looking like the original game.
Same goes for all the other changes graphics.

Reply:
I disagree. Having the model of the roman building for the amazon civlisation is just wrong.

Reply:
True but for me its worse to have settlers moving through buildings than to have a roman building being operated by amazons.